src/srf/boolean.cpp

```diff
@@ -272,30 +272,33 @@ static bool KeepRegion(SSurface::CombineAs type, bool opA, SShell::Class shell,
 {
     bool inShell = (shell == SShell::Class::SURF_INSIDE),
          outSide = (shell == SShell::Class::SURF_OUTSIDE),
-         inSame  = (shell == SShell::Class::SURF_COINC_SAME),
+         coincSame  = (shell == SShell::Class::SURF_COINC_SAME),
+         coincOpp  = (shell == SShell::Class::SURF_COINC_OPP),
          inOrig  = (orig == SShell::Class::SURF_INSIDE);
 
+    // This one line is not really part of this functions logic
```

Comment:
@phkahler Why do you think the line is "not really part of this functions logic"?